src/TypeScript/Analyzer/JavaScriptAnalyzer.cs

```diff
@@ -107,33 +112,38 @@ internal async Task ExecuteAnalysis(string filePath, IIssueConsumer consumer, Ca
             // Switch to a background thread
             await TaskScheduler.Default;
 
+            analysisStatusNotifier.AnalysisStarted(filePath);
+
             try
             {
                 await EnsureEslintBridgeClientIsInitialized(cancellationToken);
-
+                var analysisStartTime = DateTime.Now;
```

Comment:
I don't know why we didn't originally use a `Stopwatch` in the CFamily version, and then `Stopwatch.Elapsed` to get the accurate `TimeSpan`.  Was there a reason?

Reply:
No idea, good call.